CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Changes:
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches 
```

Comment:
I think allowing the 'why' to be stated in the PR itself is also good, this'll save us from creating issues for small things:

```suggestion
    All changes should have an issue in Github or the PR itself that describes well why and what 
    the change aims to achieve. All changes should be made in feature branches 
```

Reply:
I agree and update the branch accordingly.